src/core/renderers/SystemRenderer.js

```diff
@@ -38,7 +38,7 @@ class SystemRenderer extends EventEmitter
         // prepare options
         if (options)
         {
-            for (let i in CONST.DEFAULT_RENDER_OPTIONS)
+            for (const i in CONST.DEFAULT_RENDER_OPTIONS)
```

Comment:
the iterator can be const in `for-in` statements and `for-of` statements, just not in `for(;;)` statements.

> ForDeclaration : LetOrConst ForBinding

http://www.ecma-international.org/ecma-262/6.0/index.html#sec-for-in-and-for-of-statements-static-semantics-boundnames